tests/tck/features/lookup/EdgeIndexFullScan.feature

```diff
@@ -172,39 +211,16 @@ Feature: Lookup edge index full scan
       | 0  | Start             |              |                                                   |
 
   Scenario: Edge with relational CONTAINS/NOT CONTAINS filter
-    When profiling query:
-      """
-      LOOKUP ON edge_1 WHERE edge_1.col1_str CONTAINS toLower("L") YIELD edge_1.col1_str
-      """
```

Comment:
Delete these cases in another pr.

Reply:
This is actually not deletion but modification, and this is to adapt the change made in LOOKUP validator. It makes more sense to me to include this in this PR.